tools/plugin-toolkit/tsconfig.json

```diff
@@ -3,13 +3,12 @@
     "compilerOptions": {
         "declaration": true,
         "declarationMap": true,
+        "esModuleInterop": true,
         "target": "es5",
-        "jsx": "react",
-        "lib": ["es7", "dom"],
         "module": "commonjs",
-        "moduleResolution": "node",
-        "esModuleInterop": true,
+        "lib": ["es6", "dom"],
```

Comment:
Nitpick: since this is a CLI tool, we should not use the `"dom"` lib, it will not be available.

Reply:
i took this from the other CLI.. will check it out; i have recollection we had to add it in there for some weird reason

Reply:
removed from here; both catalog-export and mock-handling include dom in libs. that is because they depend directly on api-client-bear which uses isomorphic fetch & XHR related types from browser.

we should not need it here as the toolkit shall depend only on backend-spi